openassessment/assessment/test/test_peer.py

```diff
@@ -1506,27 +1504,24 @@ def test_too_many_assessments_counted_in_score_bug(self):
             required_graded_by
         )
 
-        # Sue grades the only person she hasn't graded yet (Bob)
+        # Sue grades the only person she hasn't graded yet (Bob), with a failing grade
         peer_api.get_submission_to_assess(sue_sub['uuid'], sue['student_id'])
         peer_api.create_assessment(
             sue_sub['uuid'],
             sue['student_id'],
-            ASSESSMENT_DICT['options_selected'],
-            ASSESSMENT_DICT['criterion_feedback'],
-            ASSESSMENT_DICT['overall_feedback'],
+            ASSESSMENT_DICT_FAIL['options_selected'],
+            ASSESSMENT_DICT_FAIL['criterion_feedback'],
+            ASSESSMENT_DICT_FAIL['overall_feedback'],
             RUBRIC_DICT,
             required_graded_by
         )
 
         # This used to create a second assessment,
         # which was the bug.
-        peer_api.get_score(bob_sub['uuid'], requirements)
+        score = peer_api.get_score(bob_sub['uuid'], requirements)
 
-        # Get the assessments used to generate the score
-        # Only the first assessment should be used
-        scored_assessments = peer_api.get_assessments(bob_sub['uuid'], scored_only=True)
-        self.assertEqual(len(scored_assessments), 1)
-        self.assertEqual(scored_assessments[0]['scorer_id'], tim['student_id'])
+        # Verify that only the first assessment was used to generate the score
+        self.assertEqual(score['points_earned'], 14)
```

Comment:
Instead of "get the assessment used for scoring, and make sure it was Tim's", I gave Tim and Sue's assessments of Bob different point values so we can ensure that the score is calculated correctly.